src/applications/disability-benefits/all-claims/utils/form0781.js

```diff
@@ -23,3 +24,19 @@ export function showForm0781Pages(formData) {
     )
   );
 }
+
+/**
+ * Checks if
+ * 1. modern 0781 pages should be showing
+ * 2. the option to complete the online form is selected
+ *
+ * @param {object} formData
+ * @returns {boolean} true if COMPLETE_ONLINE_FORM is selected, false otherwise
+ */
+export function isCompletingForm0781(formData) {
+  return (
+    showForm0781Pages(formData) &&
+    formData['view:mentalHealthWorkflowChoice'] ===
+      form0781WorkflowChoices.COMPLETE_ONLINE_FORM
```

Comment:
@NB28VT, I based this off of your changes in [dc27a78](https://github.com/department-of-veterans-affairs/vets-website/pull/33655/commits/dc27a78ca4943e594109738658624a480ba67397)